Article/Article.js

```diff
@@ -1,29 +1,96 @@
+/* eslint-disable max-len */
 // Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class.
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.textContent = '-----';
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', this.expandArticle.bind(this));
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    // this.domElement.classList.toggle('article-open');
+    if (this.domElement.style.height === '400px') {
+      TweenMax.to(this.domElement, 0.5, { height: 50, ease: Power3.easeOut });
+    } else {
+      TweenMax.to(this.domElement, 0.5, { height: 400, ease: Power1.easeOut });
+    }
```

Comment:
Also with dynamic content its probably a good idea not to use set pixels for the height. I'm not overly familar with GSAP but this seems to work:

```
TweenMax.set(this.domElement, { height: 'auto '});
TweenMax.from(this.domElement, 0.5, { height: 0, ease: Power1.easeOut });
```